src/SonarScanner.MSBuild.Shim/AdditionalFilesService.cs

```diff
@@ -113,23 +110,27 @@ private static AdditionalFiles PartitionAdditionalFiles(List<string> allFiles, A
     }
 
     private static string[] GetTestExtensions(AnalysisProperties properties) =>
-        properties
-            .Where(x => SupportedTestLanguages.Contains(x.Id))
-            .SelectMany(x => x.Value.Split(Comma))
-            .SelectMany(x => SupportedTestInfixes.Select(infix => $"{infix}{x.Trim()}$"))
-            .Distinct()
-            .ToArray();
+        properties is null
+            ? []
+            : properties
+                .Where(x => SupportedTestLanguages.Contains(x.Id))
```

Comment:
This one also needs a StringComparison specified. I think it is safe to use OrdinalIgnoreCase everywhere (also on Unix because the extension casing shouldn't matter there as well).

Reply:
What do you mean here?
This is comparing the property, e.g. `sonar.javascript.file.suffixes`, 
I don't think it can be different casing.

Reply:
You are right. This isn't needed here but on the `GetTestExtension`call site: https://github.com/SonarSource/sonar-scanner-msbuild/pull/2106/files#r1689996064